.github/workflows/nodejs.yml

```diff
@@ -18,22 +18,22 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [8.x, 10.x]
+        node-version: [10.x, 12.x, 13.x, 14.x]
```

Comment:
Little point doing `13.x`, it'll never be LTS. The odd numbered releases are like "preview" releases, while the even numbers are like "release candidate" before they're marked for LTS.